### PR TITLE
Add a Latest-Packages for rc3 \ preview2

### PR DESCRIPTION
**Pull request by pranavkm**
Stop gap until https://github.com/dotnet/cli/issues/2999 is sorted.

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @eerhardt

File: build-info/dotnet/cli/master/Latest_Packages.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Microsoft.DotNet.Cli.Utils 1.0.0-preview2-002765
+Microsoft.DotNet.Compiler.Common 1.0.0-preview2-002765
+Microsoft.DotNet.Files 1.0.0-preview2-002765
+Microsoft.DotNet.ProjectModel.Loader 1.0.0-preview2-002765
+Microsoft.DotNet.ProjectModel.Workspaces 1.0.0-preview2-002765
+Microsoft.Extensions.Testing.Abstractions 1.0.0-preview2-002765
+Microsoft.DotNet.InternalAbstractions v1.0.0-rc3-002765
```
Comment (eerhardt) [inline build-info/dotnet/cli/master/Latest_Packages.txt]: Don't need `v` in `v1.0.0-rc3-002765`

**Comment by eerhardt** [comment]
What's the expectation of keeping this up-to-date until https://github.com/dotnet/cli/issues/2999 is finished? Will it be manual?

**Comment by pranavkm** [comment]
Something along those lines. Picking up rc2 packages is a bit problematic for us right now since there is a higher version on NuGet.org.

File: build-info/dotnet/cli/master/Latest_Packages.txt (hunk 1 above)
Comment (pranavkm) [inline build-info/dotnet/cli/master/Latest_Packages.txt]: Updated.

**Pull request merged**